tests/foreman/cli/test_docker.py

```diff
@@ -37,50 +37,101 @@
 from robottelo.config import settings
 from robottelo.constants import DOCKER_REGISTRY_HUB
 from robottelo.constants import DOCKER_RH_REGISTRY_UPSTREAM_NAME
-from robottelo.datafactory import generate_strings_list
+from robottelo.constants import DOCKER_UPSTREAM_NAME
+from robottelo.constants import REPO_TYPE
 from robottelo.datafactory import invalid_docker_upstream_names
+from robottelo.datafactory import parametrized
 from robottelo.datafactory import valid_docker_repository_names
 from robottelo.datafactory import valid_docker_upstream_names
 from robottelo.decorators import skip_if_not_set
-from robottelo.test import CLITestCase
-from robottelo.vm import VirtualMachine
 
-DOCKER_PROVIDER = 'Docker'
-REPO_CONTENT_TYPE = 'docker'
-REPO_UPSTREAM_NAME = 'busybox'
+BASE_URL = 'http://download.eng.rdu2.redhat.com/rhel-7/rel-eng'
+SERVER_REPO = f'{BASE_URL}/updates/RHEL-7/latest-RHEL-7.9/compose/Server/x86_64/os/'
+OPTIONAL_REPO = f'{BASE_URL}/RHEL-7/latest-RHEL-7/compose/Server-optional/x86_64/os/'
+EXTRAS_REPO = f'{BASE_URL}/EXTRAS-7/latest-EXTRAS-7.9-RHEL-7/compose/Server/x86_64/os/'
```

Comment:
Immediately remove them from here as we are exposing RedHats internal info here. Ask these repos from repos.yaml OR robottelo section in robottelo.properties !